env_config.rb

```diff
@@ -31,6 +31,7 @@
     mandatory :CDN_ASSETS_DISTRIBUTION_ID, :string
     mandatory :REGISTRATION_QUEUE, :string
     mandatory :LIVE_QUEUE, :string
+    optional :DEVELOPMENT_OFFLINE_MODE, :bool, false
```

Comment:
You shouldn't need this. A variable introduced under this `if Rails.env.production?` branch that has `DEVELOPMENT` in its name is very confusing

Reply:
You could change your call-site to `unless Rails.env.development? && EnvConfig.OFFLINE?` to make sure you never query a non-existing variable.

Reply:
We already have several other use-cases of this pattern, see the block of definitions close to `DISABLE_BULLET`. There are 5 or so variables over there that simply "do not exist" in prod mode, and your new one should definitely become one of them.

Reply:
Okay, changed. I didn't new we can have a env just for development. Thanks for that.